From 5917872c144260bd6d74f35b6dadf0a98a53df07 Mon Sep 17 00:00:00 2001
From: Andy Green <andy@openmoko.com>
Date: Wed, 2 Jul 2008 22:39:25 +0100
Subject: [PATCH] fix-jbt6k74-force-reset-suspend.patch
 Sean McNeil reports that without the reset for LCM in suspend, he
 gets a monochromatic mode on it somehow.
 Signed-off-by: Andy Green <andy@openmoko.com>

---
 drivers/video/display/jbt6k74.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/video/display/jbt6k74.c b/drivers/video/display/jbt6k74.c
index 8e7bf36..7cc00ad 100644
--- a/drivers/video/display/jbt6k74.c
+++ b/drivers/video/display/jbt6k74.c
@@ -638,7 +638,7 @@ static int jbt_suspend(struct spi_device *spi, pm_message_t state)
 
 	jbt->have_resumed = 0;
 
-//	(jbt6k74_pdata->reset)(0, 0);
+	(jbt6k74_pdata->reset)(0, 0);
 
 	return 0;
 }
-- 
1.5.6.5

